tests/src/test/scala/scalaz/MaybeTests.scala

```diff
@@ -43,6 +43,7 @@ object MaybeTest extends SpecLite {
   checkAll(align.laws[Maybe])
   checkAll(equal.laws[Maybe[Int]])
   checkAll(band.laws[Maybe[ISet[Int]]])
+  checkAll(semilattice.laws[Maybe[ISet[Int]]])
```

Comment:
We can remove `band.laws[Maybe[ISet[Int]]]` if `trait Semilattice extends Band` and `semilattice laws` include `band laws`